pkg/workloadmeta/collectors/internal/containerd/image_sbom_trivy.go

```diff
@@ -98,15 +98,23 @@ func (c *collector) startSBOMCollection(ctx context.Context) error {
 				log.Warnf("Unable to close trivy client: %v", err)
 			}
 		}()
+		cleanTicker := time.NewTicker(config.Datadog.GetDuration("container_image_collection.sbom.cache_clean_interval") * time.Second)
```

Comment:
`GetDuration(…)` already returns a `time.Duration`. So, we shouldn’t multiply its return value by `time.Second` as we would with an `int` in order to turn it into a `time.Duration`.

```suggestion
		cleanTicker := time.NewTicker(config.Datadog.GetDuration("container_image_collection.sbom.cache_clean_interval"))
```